### DISCUSSION TITLE
Google Assistant

### DISCUSSION THREAD

**Discussion by martin22cph** [Thomas55555/husqvarna_automower#73] [Ideas]
Thanks for the great work. I know Husqvarna has officiel support for controling the Mower via Google Assistant, but only in English. I use this fantastisk Plugin to Home Assistant and from there Connected it to Google Home and can now tell to "Stop" in my native lang that is Danish, under the name i have given it "Robo Lars." But i can't start it? I can say "Hey Google, Stop Robo Lars" and i will Stop find the wire and go to the docking station." If I say "Hey Google, start Robo Lars" it says sometimes Starting Robo Lars" nothing happends, I also tryid Activate Robo Lars, and it says starting Robo Lars" = Nothing happends - and i was wondering if this is even posible via the API ? When it's on a planed time scheule - could be cool also being able to Start it, and not just Stop it :)

**Comment by Thomas55555** [comment]
I think the problem is, that the start command only works during the times in the schedule. See #58
So you may need to have a schedule for 24/7 and always start the mower. Or you can find a voice command for the "husqvarna_automower.start_and_park" service or for a script like this: https://community.home-assistant.io/t/husqvarna-automower-and-gardena-smart-system/118130/31?u=thomas55555
